imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -489,18 +491,21 @@ function orderToSalesInvoice(order) {
   const currencyCode = company.currency;
   const orderShipping = order.getShippingTotal();
   const orderDate = moment(order.createdAt).format();
-  const lineItems = order.items.map((item) => {
+  const lineItems = order.items.reduce((items, item) => {
```

Comment:
@spencern and I paired on this for a bit, and decided to go with `reduce` because we can then filter out any `undefined` results that were occurring with the map, creating a dense array with pure values only, not a sparse array with values and `undefined`'s.

Prior to this update, if the condition `items.variants.taxable` was not being met, nothing was returning. This new linter rule requires a `return`, which would mean we'd be returning `undefined` for all items where the if statement condition wasn't met.

By reducing it instead, we are eliminating those `undefined` items.

Reply:
Makes sense to me